### PR TITLE
Use updated vap schema

### PR DESCRIPTION
**Pull request by maxsmythe**
This updates the prototype VAP engine to use the new schema defined in https://docs.google.com/document/d/1FgwMQHEzKaBMQzsbWkBfuUiNcALOKyJJ3_WkgP-L7mo/edit

### CODE REVIEW

File: constraint/pkg/client/drivers/k8scel/driver.go [hunk 1]
```diff
@@ -48,69 +51,77 @@ import (
...
 		break
 	}
-	if rawCode == nil {
+	if source == nil {
 		return errors.New("K8sNativeValidation code not defined")
 	}
 
-	validatorCode := &admissionv1alpha1.ValidatingAdmissionPolicy{}
-	if err := runtime.DefaultUnstructuredConverter.FromUnstructuredWithValidation(rawCode, validatorCode, true); err != nil {
+	// FRICTION: Note that compilation errors are possible, but we cannot introspect to see whether any
```
Comment (ritazh) [inline constraint/pkg/client/drivers/k8scel/driver.go:95]: :(

File: constraint/pkg/client/drivers/k8scel/schema/schema.go [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+
+const (
+	// Name is the name of the driver.
+	Name           = "K8sNativeValidation"
+	ReservedPrefix = "g8r-"
+)
+
+var (
+	ErrBadType      = errors.New("Could not recognize the type")
+	ErrMissingField = errors.New("Rego source missing required field")
```
Comment (ritazh) [inline constraint/pkg/client/drivers/k8scel/schema/schema.go:24]:
```suggestion
	ErrMissingField = errors.New("K8sNativeValidation source missing required field")
```

Comment (maxsmythe) [reply constraint/pkg/client/drivers/k8scel/schema/schema.go]: Fixed

File: constraint/pkg/client/drivers/k8scel/driver.go (hunk 1 above)
Comment (maxsmythe) [reply constraint/pkg/client/drivers/k8scel/driver.go:95]: Yeah, K8s will probably need to fix that themselves once they have status reporting.

Comment (maxsmythe) [reply constraint/pkg/client/drivers/k8scel/driver.go:95]: When they add status reporting, I mean

**Review by ritazh — APPROVED** [review]
LGTM

**Comment by maxsmythe** [comment]
Thanks!

**Pull request merged**